src/app/routes/article/getInitialData/index.ts

```diff
@@ -53,6 +61,14 @@ export default async ({
 
     const agent = certsRequired(pathname) ? await getAgent() : null;
 
+    // Need to check the page type - if IDX or FIX then redirect to a Topic Page
+    if ([FEATURE_INDEX_PAGE, IDX_PAGE].includes(json?.data?.metadata.type)) {
```

Comment:
This may mean all FIX fixture data in Simorgh needs to be updated to be in Topic data format...